Flick/Controllers/LoginViewController.swift

```diff
@@ -19,7 +19,7 @@ class LoginViewController: UIViewController {
     private let logoImageView = UIImageView()
 
     // MARK: - Private Data Vars
-    private let profileSize = CGSize(width: 50, height: 50)
+    private let profileSize = CGSize(width: 100, height: 100)
```

Comment:
increasing the size might would make the image larger and base64 string even longer right

Reply:
wonder if that's slow things down

Reply:
Yea.. right now the quality is so bad tho